src/Renderer/Renderer.fs

```diff
@@ -169,35 +175,41 @@ let editMenu dispatch =
             [| // makeElmItem "Save Sheet" "CmdOrCtrl+S" (fun () -> ())
                makeElmItem "Copy" "CmdOrCtrl+C" (fun () -> dispatch Sheet.KeyboardMsg.CtrlC)
                makeElmItem "Paste" "CmdOrCtrl+V" (fun () -> dispatch Sheet.KeyboardMsg.CtrlV)
+               menuSeparator
+               makeElmItem "Rotate Left" "CmdOrCtrl+L" (fun () -> rotateDispatch Sheet.RotateMsg.Left)
+               makeElmItem "Rotate Right" "CmdOrCtrl+R" (fun () -> rotateDispatch Sheet.RotateMsg.Right)

```

Comment:
```suggestion
               makeElmItem "Rotate Left" "CmdOrCtrl+Q" (fun () -> rotateDispatch Sheet.RotateMsg.Left)
               makeElmItem "Rotate Right" "CmdOrCtrl+E" (fun () -> rotateDispatch Sheet.RotateMsg.Right)

```

Reply:
Happy to change that. We have a long way to go before the team9 changes go back to master: it will include, for menus and keys, adding vertical flip, and making sure keys don't affect open properties text boxes. Therefore key bindings may change again before we are finished.

Reply:
However, I can't do this merge yet because I have not yet brought the team9 branch uptodate with master. I will leave it open and do the merge from master now, first.

Reply:
ok, done merge (conflicts resolved)